pkg/api/types.go

```diff
@@ -1008,6 +1008,20 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+
+	// SupplementalGroups can be used to specify a list of
+	// additional groups which the main container process will run
+	// as. This will be applied to all containers in the pod in
+	// addition to the primary group of the container.
+	SupplementalGroups []int64 `json:"supplementalGroups,omitempty"`
+
+	// FSGroup is a special supplemental group that all containers in a pod
+	// run as.  If FSGroup is set, the kubelet will make the volumes the pod
```

Comment:
this is one broken sentence.  "the kubelet will make the volumes the pod has exclusive ownership of owned by this group"